engines/python/setup/djl_python/properties_manager/hf_properties.py

```diff
@@ -120,14 +121,15 @@ def construct_kwargs_device_map(self):
             self.device = None
             logging.info(f"Using device map {self.device_map}")
         elif self.tensor_parallel_degree > 0 \
+                and self.pipeline_parallel_degree > 0 \
                 and self.cluster_size > 0 \
                 and torch.cuda.device_count() > 0:
             self.kwargs["device_map"] = "auto"
             self.device = None
             world_size = torch.cuda.device_count() * self.cluster_size
-            assert world_size == self.tensor_parallel_degree, \
-                f"TP degree ({self.tensor_parallel_degree}) doesn't match available GPUs ({world_size})"
-            logging.info(f"Using {world_size} gpus")
+            assert world_size == self.tensor_parallel_degree*self.pipeline_parallel_degree, \
```

Comment:
avoid use `assert` in production code:

```
            if world_size != self.tensor_parallel_degree * self.pipeline_parallel_degree:
                raise ValueError(
                    f"TP*PP degree ({self.tensor_parallel_degree*self.pipeline_parallel_degree}) doesn't match available GPUs ({world_size})"
                )
```

Reply:
I did not add this assert previously, just modified it. I have now removed it to use ValueError